cnxarchive/views.py

```diff
@@ -115,7 +115,7 @@ def is_latest(id, version):
 
 
 TYPE_INFO = []
-LEGACY_EXTENSION_MAP = {'epub': 'epub', 'pdf': 'pdf', 'zip': 'complete.zip'}
+LEGACY_EXTENSION_MAP = {'epub': 'epub', 'pdf': 'pdf', 'zip': 'offline.zip'}
```

Comment:
I though we consciously made the choice to serve the complete, not the offline, since the offline does not actually work for what it claims to be (viewable offline) - in fact it fails to build in some cases, and is built _from_ the complete zip, so complete zip should exist wherever offline zip does. Is there a bug number that references this?